src/main/resources/instructorEmailFragment-reminderToJoinCourseBeforeSubmittingResponse.html

```diff
@@ -0,0 +1,29 @@
+<p>
+    Note that you will need to join the course as an instructor
+    before you can ${feedbackAction} your feedback for this session.
+</p>
+
+<p>
+    Below is an extract of the original email sent to you earlier,
+    which contains instructions on how to join the course.
+    To contact the person who invited you to the course, please revisit the original email.
+</p>
+<!-- some code duplication here with instructorEmailFragment-courseJoin.html, not sure if should abstract out
```

Comment:
we can remove this comment, code duplication is understandable. The inviter information is not stored because there is no need to store it.

Reply:
Hi @rrtheonlyone, I've fixed the conflicts and removed this comment. I realised I overlooked tests for this pr so I added a few lines into EmailGeneratorTest.java. please have a look before merging thank you!